packages/react-core/src/components/Banner/Banner.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Banner/banner';
+import { css } from '@patternfly/react-styles';
+
+export interface BannerProps extends React.HTMLProps<HTMLButtonElement> {
+  /** Content rendered inside the banner */
+  children?: React.ReactNode;
+  /** Additional classes added to the banner */
+  className?: string;
+  /** Variant styles for the banner */
+  variant?: 'default' | 'info' | 'danger' | 'success' | 'warning';
+  /** If set to true, the banner sticks to the top of its container */
+  isSticky?: boolean;
+}
+
+const variantStyle = {
```

Comment:
rather than having this const can we use the css 'getModifier` function?

Reply:
Sorry.  Did not realize that had been removed.  Let's follow what is done in Alerts.